app/build.gradle

```diff
@@ -18,7 +18,7 @@ repositories {
 android {
     compileSdkVersion 24
     //noinspection GradleDependency
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.0'
```

Comment:
Android Studio did that.Thank you for pointing out :smile:   I will correct that.